package.json

```diff
@@ -51,6 +54,11 @@
     "react-test-renderer": "18.0.0",
     "semantic-release": "^19.0.3"
   },
+  "peerDependencies": {
+    "react": ">=16.0.0",
+    "react-native": ">=0.59",
+    "react-test-renderer": ">=16.0.0"
```

Comment:
It's not optional, as Jest Native matchers all work on `ReactTestInstance` from React Test Renderer, so user needs to have it. We specify it as peer dep so that we do not cause multiple versions to be imported because then things break.

Reply:
The only import I see, is when used for types in `index.d.ts`, so where do we need it more?

Reply:
Since this is untyped JS code, so there are no imports, but we operate on ReactTestInstance object instances that are created by React Test Renderer. Hence, we have a mandatory peer dependency with RTR.

BTW this review is getting a bit out of scope, as this is a simple PR about adding ESLint to the project and fixing related errors.